wolfcrypt/test/test.c

```diff
@@ -22833,6 +22834,7 @@ int ed448_test(void)
     };
 
     static const byte* sigs[] = {sig1, sig2, sig3, sig4, sig5, sig6};
+    #define SIGSZ sizeof(sig1)
```

Comment:
There is another one like this in `ed25519_test`.  Out of curiosity what's wrong with the `sizeof(sigs[i]` used previously?

Reply:
sigs[] is an array of const char pointers. That sizeof is 8, not the desired 114.

Reply:
In ed25519_test, they didn't use sizeof() for the length of the signature values. All the XMEMCMP()s are using a hardcoded length of 64.